tasks/extract_filepaths_for_granule/test/unit_tests/testevents/task_event.json

```diff
@@ -63,6 +58,13 @@
               {"regex": ".*.cmr.json$", "sampleFileName": "L0A_HR_RAW_product_0001-of-0019.cmr.json", "bucket": "public"
             }
           ],
+        "buckets": 
+        [
```

Comment:
Not an array. This should be an object.

Reply:
changed to object. Seems like this is why I was getting the error `TypeError: list indices must be integers or slices, not str` in my unit tests .